modules/doc/content/getting_started/installation/install_libtorch.md

```diff
@@ -16,22 +20,31 @@ ldd --version
 
 !alert-end!
 
-## Setting up the environment
+## Setup Environment
+
+For both Mac and Linux workstations, the user needs to follow the
+[Conda MOOSE Environment](installation/conda.md) instructions.
+
+## Install Libtorch
+
+The user can choose from three alternatives when it comes to installing `libtorch`:
+
+- +Install using conda:+
 
-For Mac workstations, the user needs to create a conda environment using the
-instructions [here](installation/conda.md). On Linux machines, however,
-we cannot use the conda packages due to the mismatch between `libc` versions.
-For this reason, given that the system `libc` version allows the linking between
-the two libraries, we need to install `PETSc` and `libMesh` manually. For instructions
-on the installation of these, see [installation/hpc_install_moose.md].
+  For ARM and Intel Mac workstations the user can install libtorch using conda within the `moose` environment:
 
-In situations when the `libc` version allows the linking but the compiler stack
-has been compiled with an older `libc` version (HPC machines potentially), we need to build the
-compiler from scratch. For instructions in this process, visit [installation/manual_installation_gcc.md]
+  ```bash
+  conda install pytorch
+  ```
 
-## Installing Libtorch
+  This will provide the headers and libraries needed to use libtorch.
 
-The user can choose from two alternatives when it comes to installing `libtorch`:
+  !alert! note
+
+  The same process works for Linux workstations if the user requires CPU support only.
+  For notes on the GPU support, visit the [notes below](install_libtorch.md#gpu-notes).
```

Comment:
Can you add the command for linux cpu pytorch somewhere on this page as well? The command would be
```
conda install pytorch cpuonly -c pytorch
```
The `cpuonly` package is a dummy package from the `pytorch` channel that tells conda to pick the cpu version.

Reply:
Alternatively, just point the user to the pytorch official installation page https://pytorch.org/get-started/locally/ and follow the conda instructions, which might be better in case pytorch updates their instructions.

Reply:
Yeah that is a great catch. I think we will have to create a CIVET test for all of these. We almost jumped over this fence without it.